grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# Lint as: python3
+import os
+import json
+import typing
+from typing import Any, List, Tuple, Dict, Text
+
+from werkzeug.wrappers import Request, Response
+from werkzeug.wrappers.json import JSONMixin
+from werkzeug.routing import Map, Rule
+from werkzeug.exceptions import HTTPException, NotFound
+from werkzeug.middleware.shared_data import SharedDataMiddleware
+from werkzeug.utils import redirect
+from werkzeug.serving import run_simple
+
+from absl import app
+from absl import flags
+
+from grr_response_core.config import server as config_server
+from grr_response_server import fleetspeak_connector
+from grr_response_server import fleetspeak_utils
+from grr_response_server import server_startup
+from grr_response_core import config
+from grr_response_core.config import contexts
+
+from fleetspeak.src.server.proto.fleetspeak_server.resource_pb2 import ClientResourceUsageRecord
+
+
+AVAILABLE_METRICS = ['mean_user_cpu_rate', 'max_user_cpu_rate', 'mean_system_cpu_rate', 
+                    'max_system_cpu_rate', 'mean_resident_memory_mib', 'max_resident_memory_mib']
```

Comment:
I would keep this list as a "listing", i.e.:
```
FOO = [
    "foo",
    "bar",
    "baz",
]
```
(note that the trailing comma, it is needed to prevent the auto-formatter from folding it)